module_14/tests/demoblaze_test.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from module_14.pages.home_page import HomePage
+from module_14.pages.product_page import ProductPage
+from module_14.pages.cart_page import CartPage
+
+
+@pytest.mark.usefixtures("setup")
+class TestDemoBlaze:
+
+    def test_login(self):
+        home_page = HomePage(self.driver)
+        home_page.open_home_page()
+        home_page.click_login_button()
+        home_page.log_in("snowfallslow@gmail.com", "1qa2ws3ed")
+        home_page.check_log_out_button_is_visible()
+        home_page.validate_welcome_message_text("snowfallslow@gmail.com")
+
+    def test_add_to_cart(self):
```

Comment:
this test has a specific precondition User is logged in that was intended to be a fixture

Reply:
Thank you for this comment, Anastasiia! Now I understand how to pass some object from one fixture to another:)